### DISCUSSION TITLE
encoding profiles

### DISCUSSION THREAD

**Discussion by mohmafia** [mediacms-io/mediacms#309] [Q&A]
I got mediacms working GREAT !!
but where do i change the ffmpeg profiles like scaling filters and bitrate 
example ffmpeg -i -vf scale=850x478 -aspect 16:9 -movflags +faststart -preset faster -vcodec h264 -profile:v high10 -level:v 5.1 -crf 8 -maxrate 2.6M -bufsize 1150K -tune fastdecode -acodec aac -ar 44100 -b:a 170k  -filter:a "volume=2.1" -threads 10  -y
thanks

**Comment by mgogoulos** [comment]
Hi, moving this to discussions. There is not a single place that the ffmpeg commands are defined, but you can have a look on files/tasks.py and files/helpers.py , where the ffmpeg commands are produced after figuring out basic information about the video dimensions. If you are not planning to do radical changes to the existing profiles, I believe you will be able to adjust existing settings with the ones you prefer. I am also working on the documentation of this part so there are more hints as far as what is happening...Hope this helps

**Reply by mohmafia** [reply]
ty ... there is also the problem that profiles bigger then 720P does not encode it fails and is stuck on pending ... with htop i can see the encoding gets called but ffmpeg exits in 2 seconds after being called that suggests that there is somthing wrong in the command that gets passed  as far as i see and tested this is not really ready for on a production server if it did not had these issues i would love to use it and categories aint working there is no where to selct the category the video needs to be in and a very important feature is also missing renaming of a video .. that is a very basic function that should be in every video cms and a netflix style template would be welcome so you can list videos by categories on the front page like like WWBN does which i use at this moment

**Reply by mohmafia** [reply]
let me show you some examples.....
front page that list video's by category
![image](https://user-images.githubusercontent.com/61617857/137834886-c2d95a69-f6fd-4e16-852d-7d549ef6f937.png)
easy way to change ffmpeg settings for encoding
![image](https://user-images.githubusercontent.com/61617857/137835058-aac34db6-e4a8-48be-9607-cc49a2967cbb.png)
